### DISCUSSION TITLE
import in latest Next.js (hydration error)

### DISCUSSION THREAD

**Discussion by adekpp** [wojtekmaj/react-calendar#889] [Ideas]
`import Calendar from "react-calendar"` - Causes hydration error.

Next.js docs: 

> Next.js allows you to [disable prerendering](https://nextjs.org/docs/app/building-your-application/optimizing/lazy-loading#skipping-ssr) on specific components, which can prevent hydration mismatches.

So the import can look like this: `const Calendar = dynamic(() => import("react-calendar"), {
  ssr: false,
});`

**Comment by wojtekmaj** [comment]
What's the error? Which is the latest version working for you?
